cwel/component/button/button.scss

```diff
@@ -0,0 +1,136 @@
+////
+/// @group _button.scss
+/// @since 0.0.1
+////
+
+/// 1. removes focus around text in the button tag firefox
+
+button {
+    width: 100%;
+    color: $c_secondary;
+    font-family: inherit;
+    font-size: inherit;
+    background-color: transparent;
+    outline: none;
+    border: none;
+
+    &:focus { outline: none; }
+    &::-moz-focus-inner { border: 0; } // 1
+
+    @include breakpoint('md') { width: auto; }
+}
+
+/// 1. Show no outline as the focus style is defined on the border
+/// 2. Inline-flex on medium and above so the button doesn't go full-width but keeps the properties of flex
+
+.button {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 56px; // stylelint-disable-line
+    min-width: 300px; // stylelint-disable-line
```

Comment:
@NathanRodger I'd place a comment above this line to explain the need for `stylelint-disable-line`